tutorial/advanced/t13_multi-dataset_training_evaluation.ipynb

```diff
@@ -0,0 +1,502 @@
+      "FastEstimator-Train: step: 200; ce: 0.09880432; steps/sec: 156.39;\n",
+      "FastEstimator-Train: step: 200; epoch: 1; epoch_time: 2.23 sec;\n",
+      "FastEstimator-Eval: step: 200; epoch: 1; accuracy: 0.8439442717584369; accuracy|mnist: 0.9562; accuracy|svhn: 0.8008220651505839; ce: 0.5176362;\n",
+      "FastEstimator-Train: step: 300; ce: 0.25202054; steps/sec: 94.94;\n",
+      "FastEstimator-Train: step: 400; ce: 0.059333216; steps/sec: 170.28;\n",
+      "FastEstimator-Train: step: 400; epoch: 2; epoch_time: 1.64 sec;\n",
+      "FastEstimator-Eval: step: 400; epoch: 2; accuracy: 0.905833703374778; accuracy|mnist: 0.9787; accuracy|svhn: 0.8778426551936078; ce: 0.31812456;\n",
+      "FastEstimator-Train: step: 500; ce: 0.035844967; steps/sec: 97.73;\n",
+      "FastEstimator-Train: step: 600; ce: 0.067981; steps/sec: 170.26;\n",
+      "FastEstimator-Train: step: 600; epoch: 3; epoch_time: 1.61 sec;\n",
+      "FastEstimator-Eval: step: 600; epoch: 3; accuracy: 0.9265930284191829; accuracy|mnist: 0.9853; accuracy|svhn: 0.904041180086048; ce: 0.25179437;\n",
+      "FastEstimator-Train: step: 700; ce: 0.5206413; steps/sec: 96.37;\n",
+      "FastEstimator-Train: step: 800; ce: 0.0029142646; steps/sec: 172.17;\n",
+      "FastEstimator-Train: step: 800; epoch: 4; epoch_time: 1.62 sec;\n",
+      "FastEstimator-Eval: step: 800; epoch: 4; accuracy: 0.9384436056838366; accuracy|mnist: 0.9892; accuracy|svhn: 0.9189459127228027; ce: 0.21141718;\n",
+      "FastEstimator-Finish: step: 800; model_lr: 0.001; total_time: 38.69 sec;\n"
+     ]
+    }
+   ],
+   "source": [
+    "from fastestimator.trace.io import BestModelSaver\n",
+    "from fastestimator.trace.metric import Accuracy\n",
+    "\n",
+    "\n",
+    "estimator = fe.Estimator(pipeline=pipeline,\n",
+    "                         network=network,\n",
+    "                         epochs=4,\n",
+    "                         traces=Accuracy(true_key=\"y\", pred_key=\"y_pred\"),\n",
+    "                         max_train_steps_per_epoch=200)\n",
+    "\n",
+    "estimator.fit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see in the training log, the `Accuracy` Trace created 3 keys: `accuracy|mnist`, `accuracy|svhn`, and `accuracy`.  The `accuracy|mnist`, `accuracy|svhn` are measured on individual dataset, and `accuracy` is measured on overall combined evaluation set combined."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Customize AUC metric that works for every dataset, then save model based on AUC of specific dataset\n",
+    "\n",
+    "Since this is a 10-class classification task, to simply AUC calculation, we will count any label < 5 as 0, the rest labels as 1.  When a trace is initializing the data during `on_epoch_begin` and outputting the data during `on_epoch_end`, then all we need is a traceable decorator."
```

Comment:
```suggestion
    "Since this is a 10-class classification task, to simplify AUC calculation, we will count any label < 5 as 0 and the rest of labels as 1.  When a trace is initializing the data during `on_epoch_begin` and outputting the data during `on_epoch_end`, we only need a `per_ds` decorator to enable multi-dataset support as shown below."
```